sdk/nodejs/runtime/index.ts

```diff
@@ -20,7 +20,6 @@ export {
 }  from "./closure/serializeClosure";
 
 export { CodePathOptions, computeCodePaths } from "./closure/codePaths";
-export { leakedPromises } from "./debuggable";
```

Comment:
This line was added in #5347, but it's not necessary as automation API accesses `leakedPromises` internally directly from `debuggable`:

https://github.com/pulumi/pulumi/blob/dc7f3f0c66de38f6b7b0a0afb1736930683b89d6/sdk/nodejs/automation/server.ts#L52